drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/widget/analysis/DecisionTableAnalyzerTest.java

```diff
@@ -230,6 +235,8 @@ public void testConditionsShouldNotBeIgnored() throws Exception {
 
         fireUpAnalyzer();
 
+        assertContains( "EmptyRule", analyzerProvider.getAnalysisReport(), 2 );
+        assertDoesNotContain( "EmptyRule", analyzerProvider.getAnalysisReport(), 1 );
```

Comment:
This assertion is failing. Waiting for incoming PR by @Rikkola that may fix this fail.